entry.go

```diff
@@ -300,9 +378,13 @@ func (te *TunaEntry) listenTCP(ip net.IP, ports []int) ([]int, error) {
 					time.Sleep(time.Second)
 					continue
 				}
-
-				go Pipe(stream, conn, &te.bytesOut)
-				go Pipe(conn, stream, &te.bytesIn)
+				if te.Config.Reverse {
+					go Pipe(stream, conn, &reverseBytesIn[_port])
```

Comment:
Why is `reverseBytesIn` an array?

Reply:
To calculate mutiple exit client's traffic statistics

Reply:
One TuneEntry is only for one client (exit)